src/common/talents/TALENTS_MONK.js

```diff
@@ -40,6 +39,7 @@ export default {
   LIGHT_BREWING_TALENT: { id: 196721, name: "Light Brewing", icon: "spell_brew_wheat" },
   BLACK_OX_BREW_TALENT: { id: 115399, name: "Black Ox Brew", icon: "ability_monk_chibrew" },
   GIFT_OF_THE_MISTS_TALENT: { id: 196719, name: "Gift of the Mists", icon: "ability_monk_pathofmists" },
+  SUMMON_BLACK_OX_STATUE_TALENT: { id: 115315, name: "Summon Black Ox Statue", icon: "monk_ability_summonoxstatue" },
```

Comment:
Might be a bug in the generateTalents script, but I hadn't noticed it before so maybe something weird is going on with Monks 🤔